frontend/apps/filemanager/filemanagerbookinfo.lua

```diff
@@ -269,23 +281,26 @@ function BookInfo:onShowBookInfo()
     end
 end
 
+function BookInfo:showBookProp(prop_key, prop_text)
+    if prop_key == "description" then
+        prop_text = util.htmlToPlainTextIfHtml(prop_text)
+    end
+    UIManager:show(TextViewer:new{
+        title = self.prop_text[prop_key],
+        text = prop_text,
+    })
+end
+
 function BookInfo:onShowBookDescription(description, file)
     if not description then
         if file then
-            description = BookInfo.getDocProps(nil, file).description
+            description = BookInfo.getDocProps(file).description
         elseif self.document then -- currently opened document
             description = self.ui.doc_props.description
         end
     end
-    if description and description ~= "" then
-        -- Description may (often in EPUB, but not always) or may not (rarely
-        -- in PDF) be HTML.
-        description = util.htmlToPlainTextIfHtml(description)
```

Comment:
(Was a bit sad that you removed my wildlife observation notes. But it is duplicated elsewhere, so not yet lost :)

Reply:
Sorry for that, but there is another one:
https://github.com/koreader/koreader/blob/9b00a8f30480574166ed1cf3e3c3440890148a3a/frontend/apps/filemanager/filemanagerbookinfo.lua#L113